testing/python/raises.py

```diff
@@ -197,7 +197,9 @@ class ClassLooksIterableException(Exception):
             pass
 
         with pytest.raises(
-            Failed, match="DID NOT RAISE <class 'raises.ClassLooksIterableException'>"
+            # NOTE with xdist: "DID NOT RAISE <class 'raises.TestRaises.test_raises_exception_looks_iterable.<locals>.ClassLooksIterableException'>" ?!  # noqa: E501
+            Failed,
+            match=r"DID NOT RAISE <class 'raises(\..*)*ClassLooksIterableException'>",
```

Comment:
Why this change, only with xdist?

Reply:
Not related to this PR, but also with merge-master-into-features: https://travis-ci.org/pytest-dev/pytest/jobs/465815039#L570